packages/cli-repl/src/logger.ts

```diff
@@ -38,217 +40,217 @@ function logger(bus: any, logDir: string) {
 
   bus.on('method:aggregate', function(collection, pipeline) {
     const params = { collection, pipeline };
-    log.info('method:aggregate', params);
+    log.info('method:aggregate', redactInfo(params));
```

Comment:
We do `redactInfo(params)` everywhere, maybe we can extract this in a function:

```
function logMethodCall(method, params) {
   log.info(method, redactInfo(params));
}
```

This way next time we have to do a change in format is going to be just one place.